_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -23,15 +23,19 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to each [link history state][] of any [semantic link][], for which all the following is true:
+This rule applies to any [semantic link][], for which all the following is true:
 
 - **link text**: the [semantic link][] element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
 - **non-link line text**: the [semantic link][] element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
 - **different hue**: the [foreground colors][foreground color] of the [semantic link][] and the [foreground color][] of the **non-link line text** elements have a [different hue][], or the [background colors][background color] of the [semantic link][] and the [background color][] of the **non-link line text** elements have a [different hue][].
 
-## Expectation
+## Expectation 1
 
-For each target, when its [semantic link][] is not in the [focused][] state and the [hovered][] state at the same time, there exists at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**.
+For each test target, there exists at least one [inspection state][] that is not [focused][] and [hovered][] at the same time and at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**.
+
+## Expectation 2
+
+If the test target matches the [`:any-link` pseudo-class](https://drafts.csswg.org/selectors-4/#any-link-pseudo), Expectation 1 must hold for each [link history state][] with a **different hue**.
```

Comment:
I'm not fully sure about that articulation. The point is that Expectation 1 depends on the tester.
If I go to Failed Example 2 without viewing the link first, I would pass Expectation 1 and fail Expectation 2. On the other hand, if I go to that page after viewing the link (the link would be `:visited` when I arrive), I would fail both Expectations.

This is maybe not a huge deal given that the end result is the same (rule fails), but still weird that the individual expectation depend on the tester.

I'm tackling the same issue a bit differently in #1455 where I need to consider even more states. Not sure this can help here…

Reply:
As you said, the end result is the same. Given that expectation 2 will always force you to inspect all link history states, unless the first one you checked immediately fails expectation 1, there is no way different testers will reach different results. 

If more people feel this is something that needs to be addressed, then I can look into it, but I don't really think it needs changing.